### PR TITLE
Sb 2.1.x upgrade

### PR DESCRIPTION
**Pull request by gytis**
I've put a bunch of commits in one PR because they're kind of related. It includes the following:
1. A fix for the integration tests, which started failing for some reason before even changing anything. Probably caused by upgrade to SB 2
2. Booster parent removal
3. Dependencies upgrade (AMQP Hub, Artemis, Spring Boot BOM)
4. AMQ template upgrade to 7.2. from 6.3

### CODE REVIEW

**Comment by gytis** [comment]
My bad, I think this will fail. I forgot to change template parameters in CI config

**Comment by geoand** [comment]
No worries, I was waiting for CI before checking anyway :)

File: pom.xml [hunk 1]
```diff
@@ -17,13 +12,28 @@
...
   <properties>
-    <amqphub.version>1.0.0</amqphub.version>
-    <artemis.version>1.5.6</artemis.version>
-    <arquillian.version>1.4.0.Final</arquillian.version>
-    <commons-logging.version>1.2</commons-logging.version>
-    <rest-assured.version>3.1.0</rest-assured.version>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```
Comment (geoand) [inline pom.xml]: :+1: We should probably do the same for the other boosters at some point.

Comment (gytis) [reply pom.xml]: I agree, the top part of these properties is applicable to all

**Pull request merged**